apps/system/test/unit/call_forwarding_test.js

```diff
@@ -170,33 +170,51 @@ suite('system/callForwarding >', function() {
         window.asyncStorage.getItem.restore();
       });
 
-      test('should early returrn if the sim card is not available', function() {
-        this.callForwarding.start();
-        sinon.spy(window.asyncStorage, 'getItem');
-        this.slots[0].simCard = null;
-        this.callForwarding._initCallForwardingState(this.slots[0]);
-        sinon.assert.notCalled(window.asyncStorage.getItem);
-        window.asyncStorage.getItem.restore();
+      test('should early return and set call forwarding to false ' +
+        'if the sim card is not available', function(done) {
+          this.callForwarding.start();
```

Comment:
`MockSettingsHelper.instances['ril.cf.enabled'].value[0]` is false by default, so I would suggest to set it to true before `start()`.